wire-ios-data-model/Source/Model/AccountStore.swift

```diff
@@ -41,7 +41,7 @@ public final class AccountStore: NSObject {
     public required init(root: URL) {
         directory = root.appendingPathComponent(AccountStore.directoryName)
         super.init()
-        FileManager.default.createAndProtectDirectory(at: directory)
+        try! FileManager.default.createAndProtectDirectory(at: directory)
```

Comment:
Should we create ticket for cleaning up all these `try!`?

Reply:
Before it has crashed inside the func with `fatal`, so I think it's better now to have it visible. But yeah please let's try to avoid these force casts via refactoring ticket 👍

Reply:
yeah I want to get rid of them but didn't want to do it in this PR. I can make a general ticket to clean up all force trys

Reply:
created a ticket https://wearezeta.atlassian.net/browse/WPB-7331